rotkehlchen/chain/evm/decoding/curve/curve_cache.py

```diff
@@ -445,11 +437,10 @@ def query_curve_data_from_chain(
     return new_pools
 
 
-def query_curve_data(
-        inquirer: 'EthereumInquirer',
-        cache_type: Literal[CacheType.CURVE_LP_TOKENS],
-) -> list[CurvePoolData] | None:
-    """Query curve lp tokens, curve pools and curve gauges.
+def get_query_curve_data(
+        chain_id: ChainID,  # pylint: disable=unused-argument  # TODO: use it for the cache keys
```

Comment:
okey you have it in the docstring that that needs to be used. I believe leaving the job half done like this is wrong because is error prone. You can create new keys with the chainId and later in the db upgrade we can rename them. I have a db upgrade in my PR so you can do the db upgrade later but I would leave the job done regarding the usage of the keys here